src/components/backend-ai-resource-policy-list.ts

```diff
@@ -67,9 +67,14 @@ export default class BackendAIResourcePolicyList extends BackendAIPage {
   @property({type: Object}) _boundPolicyNameRenderer = this.policyNameRenderer.bind(this);
   @property({type: Object}) _boundClusterSizeRenderer = this.clusterSizeRenderer.bind(this);
   @property({type: Object}) _boundStorageNodesRenderer = this.storageNodesRenderer.bind(this);
+  @query('#allowed-vfolder-hosts') private allowedVfolderHostsSelect;
+  @state() private all_vfolder_hosts;
+  @state() private allowed_vfolder_hosts;
+  @state() private default_vfolder_host = '';
```

Comment:
This state variable doesn't seem to be used anywhere.

Reply:
https://github.com/lablup/backend.ai-webui/pull/1327/files#diff-d61f09d5fee23b684f5d92bbb9987a8c9023fb98e1f639c9f4cfde7ff9d0d124R566

Reply:
The `default_vfolder_host` variable does not seem to be used in anywhere in this file.

<img width="969" alt="image" src="https://user-images.githubusercontent.com/5039744/179940243-902abfda-7f0c-4b06-8c56-69a1b9a215d5.png">